network.go

```diff
@@ -54,6 +54,10 @@ type Network interface {
 
 	// Return certain operational data belonging to this network
 	Info() NetworkInfo
+
+	// Peers returns the IP addresses of the peer nodes participating in the same overlay network
+	// For non-dynamic overlay networks and bridge networks it returns an empty slice
+	Peers() []string
```

Comment:
The `Network` interface represents the docker network concept, a set of endpoints. Besides the first class parameter (name, id and type) the rest of its members is in fact about these endpoints.

Anything else which is not part of the high level network abstraction, is one level below in the `NetworkInfo` interface: labels, ipam config and operational data, creation time and driver information.

Not sure why you say `NetworkInfo` is only for static configuration, when the description says:
`NetworkInfo returns some configuration and operational information about the network`

Peers is not part of the original network abstraction. It is not even a common concept to all kind of networks (local/global scope). It is some operational data of an internal infrastructure (networkDB) which we are exposing for the ease of debugging.

At best, it has to go inside `NetworkInfo`.